test/apps/cloudAppClient/manager-eventRequest.test.js

```diff
@@ -0,0 +1,66 @@
+'use strict'
+
+var test = require('tape')
+var helper = require('../../helper')
+var inherits = require('util').inherits
+var EventEmitter = require('events').EventEmitter
+var Manager = require(`${helper.paths.apps}/cloudappclient/manager.js`)
+var Skill = require(`${helper.paths.apps}/cloudappclient/skill`)
+var eventRequestMap = require(`${helper.paths.apps}/cloudappclient/eventRequestMap.json`)
+
+function EventRequest () {
+  EventEmitter.call(this)
+}
+inherits(EventRequest, EventEmitter)
+
+EventRequest.prototype.ttsEvent = function (name, appId, itemId, cb) {
+  var res = '{}'
+  cb(res)
+  this.emit(`tts-${appId}-${name}`, itemId)
+}
+
+EventRequest.prototype.mediaEvent = function (name, appId, extra, cb) {
+  var res = '{}'
+  cb(res)
+  this.emit(`media-${appId}-${name}`, extra)
+}
+
+test('test manager-eventRequest: media cancel', (t) => {
+  t.plan(1)
+  var skill = new Skill({}, {}, {
+    appId: 'testAppId',
+    response: {
+      action: {}
+    }
+  })
+  var eventBus = new EventRequest()
+  var pm = new Manager(null, Skill)
+  pm.eventRequest = eventBus
```

Comment:
It's better to comment something about the injection, and I'd suggest `pm._eventRequest` to represent it's private :)